cmd/alerts/common.go

```diff
@@ -0,0 +1,95 @@
+package alerts
+
+import (
+	"bytes"
+	"context"
+	"log"
+
+	"github.com/openshift/backplane-cli/cmd/ocm-backplane/login"
+	"github.com/openshift/backplane-cli/pkg/cli/config"
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/rest"
+	"k8s.io/client-go/tools/remotecommand"
+	"k8s.io/kubectl/pkg/scheme"
+)
+
+var (
+	AccountNamespace string = "openshift-monitoring"
+	ContainerName    string = "alertmanager"
+	LocalHostUrl     string = "http://localhost:9093"
+	PodName          string = "alertmanager-main-0"
+)
+
+type logCapture struct {
+	buffer bytes.Buffer
+}
+
+func (capture *logCapture) GetStdOut() string {
+	return capture.buffer.String()
+}
+
+func (capture *logCapture) Write(p []byte) (n int, err error) {
```

Comment:
This function returns an error, but it can only ever be nil. My vote would be to either

a.) Get the error from the `WriteString` call and check that
b.) not return an error value at all

Reply:
@AlexVulaj Does it looks good ?

`func (capture *logCapture) Write(p []byte) (n int, err error) {
	a := string(p)
	_, err = capture.buffer.WriteString(a)
	return len(p), err
}`